ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -34,8 +34,11 @@ - (dispatch_queue_t)methodQueue {
     [Instabug startWithToken:token invocationEvent:invocationEvent];
     RCTAddLogFunction(InstabugReactLogFunction);
     RCTSetLogThreshold(RCTLogLevelInfo);
-    [Instabug setCrashReportingEnabled:NO];
     [Instabug setNetworkLoggingEnabled:NO];
+    SEL setCrossPlatformSEL = NSSelectorFromString(@"setCrossPlatform:");
+    if ([[Instabug class] respondsToSelector:setCrossPlatformSEL]) {
+        [[Instabug class] performSelector:setCrossPlatformSEL withObject:@(true)];
```

Comment:
`@(YES)` instead of `@(true)`

Reply:
### Suggestion for future progress on this:
Take a look at `NSInvocation`, I think it's better for handling calls with parameters (like more than 1 which `performSelector` doesn't support)

**Documentation:**
https://developer.apple.com/documentation/foundation/nsinvocation

**Example:**
```
NSInvocation *inv = [NSInvocation invocationWithMethodSignature:[self.tableView methodSignatureForSelector:@selector(setEditing:Animated:)]];
[inv setSelector:@selector(setEditing:Animated:)];
[inv setTarget:self.tableView];
[inv setArgument:&yes atIndex:2]; //this is the editing BOOL (0 and 1 are explained in the link above)
[inv setArgument:&yes atIndex:3]; //this is the animated BOOL
[inv invoke]
```